routes/api.php

```diff
@@ -16,3 +16,7 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+Route::get('/courses', 'Api\CourseApi@index')->name("api.course.index");
+Route::get('/courses/{id}', 'Api\CourseApi@show')->name("api.course.show");
+Route::get('/v2/courses', 'Api\CourseApiV2@index')->name("api.v2.course.index");
```

Comment:
Only one implementation. Put api/courses/ and api/course/{id} as required and delete other routes and methods